client/components/fields/editor/base/treeSelect.tsx

```diff
@@ -16,6 +16,7 @@ export interface IEditorFieldTreeSelectProps<T = any> extends IEditorFieldProps
     valueAsString?: boolean;
     smallPadding?: boolean;
     sortable?: boolean;
+    scheme?: string;
```

Comment:
This functionality would be better placed in the `CustomVocabularies` component, allowing this component to be a simple wrapper around the UI Framework component

Reply:
any suggestion how to do it?